apps/macos/package.json

```diff
@@ -17,7 +17,8 @@
     "test": "fluentui-scripts jest",
     "macos": "react-native run-macos --scheme FluentTester --project-path src",
     "prettier": "fluentui-scripts prettier",
-    "prettier-fix": "fluentui-scripts prettier --fix true"
+    "prettier-fix": "fluentui-scripts prettier --fix true",
+    "e2etest": "wdio"
```

Comment:
Say I wanted to in the future move all the platform specific test apps into one big test app: Would it be possible to add a platform flag to this command?

Reply:
I'd have to learn a bit more about how this would work. So each platform would launch the same app? You can specify what app you want the test harness to deploy in each platforms WebDriverIO config file (wdio.conf,js)

Reply:
Yeah, I've had the idea that instead of `apps/<platform>` to run each app, there should just be one `apps/FluentTester` app that runs on all platforms that `react-native-test-app` supports (iOS, Android, macOS, Windows). I've never quite gotten it working, so it's not really on my or anyone's roadmap.